scripts/makefile/local.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
can we name this chart-local.sh ?

Reply:
service-local? Its about changing the SERVICE, Docker and Helm to local.